src/main/java/appeng/integration/modules/jei/JEIPlugin.java

```diff
@@ -89,24 +89,24 @@ private void registerDescriptions( IDefinitions definitions, IModRegistry regist
 		}
 		addDescription( registry, materials.certusQuartzCrystalCharged(), message );
 
-		if( AEConfig.instance.isFeatureEnabled( AEFeature.MeteoriteWorldGen ) )
+		if( AEConfig.instance().isFeatureEnabled( AEFeature.METEORITE_WORLD_GEN ) )
 		{
 			addDescription( registry, materials.logicProcessorPress(), GuiText.inWorldCraftingPresses.getLocal() );
 			addDescription( registry, materials.calcProcessorPress(), GuiText.inWorldCraftingPresses.getLocal() );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method appeng.integration.modules.jei.JEIPlugin.registerDescriptions(IDefinitions, IModRegistry) appears to call the same method on the same object redundantly [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=fb-contrib%3APRMC_POSSIBLY_REDUNDANT_METHOD_CALLS)